src/app.js

```diff
@@ -118,6 +118,8 @@ class SAFEApp extends EventEmitter {
                 let newPath;
                 if (!path || !path.length) {
                   newPath = '/index.html';
+                } else if (path === '/') {
+                  newPath = 'index.html';
```

Comment:
I think this is redundant with the following condition below: `else if (path[path.length - 1] === '/')`,  and it actually breaks the case of fetching something like `app.webFetch('safe://mydomain/')` (we should add this tests case, I pasted it below). I'm not sure which case you are trying to cover that is not currently covered by existing conditions.
``` javascript
  it('empty subdirectory fallback', function test() {
    this.timeout(20000);
    const content = `hello world, on ${Math.round(Math.random() * 100000)}`;
    return createRandomDomain(content, '/index.html', '')
      .then((domain) => createAnonTestApp()
        .then((app) => app.webFetch(`safe://${domain}/`)
          .then((f) => app.immutableData.fetch(f.dataMapName))
          .then((i) => i.read())
          .then((co) => should(co.toString()).equal(content))
      ));
  });
```